packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -132,40 +168,73 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 			statusComponents.push(connectionStateToString(containerState.connectionState));
 		}
 	}
-	const statusString = statusComponents.join(" | ");
+
+	function ContainerIDRow(label: string, id: string | undefined): React.ReactElement {
```

Comment:
Nit: I would probably move these outside of the `ContainerSummaryView` function scope. That way they don't accidentally capture state they shouldn't have access to.

Reply:
I don't think I can move that outside `ContainersummaryView` because it uses `columnSizing_unstable`, which is generated from the `useTableFeatures` hook inside the component.  This checks out right?  The component only has access to state it needs.

Reply:
You can pass that value down to them as props, making the dependencies explicit. As is, these are React components that depend on state that React itself is not aware of, which breaks invariants.